projects/igniteui-angular/src/lib/grids/cell.component.ts

```diff
@@ -859,6 +867,12 @@ export class IgxGridCellComponent implements OnInit, OnChanges, OnDestroy {
             return;
         }
 
+        const cellData: LayoutCellData = {
```

Comment:
Is this needed? You should be able to get the cell using the information in the selectionNode.

Reply:
I wanted it to be consistent with other cases like in the group by since there you should not get the cell since the cell might not be rendered.